help/en/html/hardware/mqtt/index.shtml

```diff
@@ -209,6 +209,14 @@
         <li><strong>Lights</strong>: JMRI sends on the "Light send topic" if a user commands a
         Light change by clicking on a panel or table. JMRI also acts on Light state messages sent
         from the layout on the "Light receive topic."</li>
+
+        <li><strong>Last Will</strong>: JMRI can specify its last will message when it connects to
+        a broker. The last will message is a normal MQTT message with a topic, retained message
+        flag, QoS, and payload. The broker stores the message until it detects that the client
```

Comment:
“The broker stores the message until its detects that the client” - does “client” in this case refer to JMRI?  That should be stated so that it is clear this only comes into play if JMRI disconnects unexpectedly.    This also requires that any layout control objects subscribe to the last will topic or they won’t be notified.

BTW, this text appears to have been copied verbatim from https://www.hivemq.com/blog/mqtt-essentials-part-9-last-will-and-testament/ which is a copyrighted site.  Do you/we have permission to copy and paste on the JMRI.org site?

Reply:
> BTW, this text appears to have been copied verbatim from https://www.hivemq.com/blog/mqtt-essentials-part-9-last-will-and-testament/ which is a copyrighted site. Do you/we have permission to copy and paste on the JMRI.org site?

Small amounts can be quoted as fair use _iff_ the source is cited.  Adding a link to where to find more info both meets that requirement and can help people who want to know more.